src/config/seo_meta.json

```
@@ -1,7 +1,7 @@
 {
-  "title": "Juicebox",
+  "title": "Juicebox | Programmable Crypto Fundraising & DAO Management",
```

```suggestion
  "title": "Juicebox | Crypto Fundraising & DAO Management",
```

Wonder if we just cut `Programmable` for the purposes of the title. Loose thought

---

Down for this. Title was acidicsantana's suggestion but I doubt "programmable" is a common search keyword.

Does feel a bit reductive. Can always come back to it. Comitting suggestion.